File: backend/src/main/java/org/jdk/project/dto/platform/PlatformBootstrapConfigDto.java
package org.jdk.project.dto.platform;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlatformBootstrapConfigDto {
  private Long id;
  private String configKey;
  private String configJson;
  private Boolean enabled;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/platform/PlatformBootstrapConfigDto.kt
package org.jdk.project.dto.platform

data class PlatformBootstrapConfigDto(
    var id: Long? = null,
    var configKey: String? = null,
    var configJson: String? = null,
    var enabled: Boolean? = null,
    var remark: String? = null,
)
